### DISCUSSION TITLE
忘记了v2rayA的用户名和密码怎么办

### DISCUSSION THREAD

**Discussion by EthanCCYX** [v2rayA/v2rayA#934] [Q&A]
### 版本、安装方式、系统

1. 你在使用什么版本的v2rayA
1.5.8.1
2. 你通过什么方式安装v2rayA
brew
3. 你所使用的操作系统
macOS Ventura
Macbook Air M1

### 描述问题：

之前安装并使用过V2raya，后来删除了文件
现在重新按照步骤安装v2ray后，进入127.0.0.1：2017后直接提示输入帐号密码
已经忘记用户名和密码了，v2raya --help只有reset password的，执行后登陆页面没有变化
我要如何重新创建用户名密码？

### v2rayA运行日志（Optional）：

 ```bash
 # daemon mode
 journalctl -u v2raya | tail -n 50
 sudo tail -n 50 /var/log/v2raya/v2raya.log

 # docker mode
 docker logs v2raya
 ```

 或者粘贴来自 Web 管理页面的日志

```shell
# 此处贴入运行日志



```

### v2rayA设置以及相关截图
贴入v2rayA设置，以及问题相关的截图。

<img width="998" alt="image" src="https://user-images.githubusercontent.com/32787955/234822453-b8c802f8-0b96-4c10-b9ec-f17ab0d13c10.png">

**Comment by DonkiChen** [comment]
`v2raya --reset-password`

**Reply by du8028934** [reply]
群晖ssh下执行没有反应。

**Comment by Ghost-in-the-Lake** [comment]
我在ubuntu中安装了v2raya，也是忘记了用户名，密码倒是记下了。卸载重装也无法进入创建新用户的界面，十分苦恼。

**Reply by EthanCCYX** [reply]
问题已解决：
请删除您用户目录下的 .config 文件夹内的 v2raya文件即可.
请注意:.config文件夹是隐藏文件. 请设置显示隐藏文件后删除.

或者在命令行中执行如下命令
sudo rm -rf ~/.config/v2raya/

**Reply by DanielDeng3** [reply]
not work for me.
i'm use brew install v2raya, deleted ~/config/v2raya and /etc/v2raya. still need past password.

**Comment by Firestar-Reimu** [comment]
你好，请问windows上安装的如何修改密码？也是忘记了账号密码，没有.config文件夹，也没有powershell命令

**Reply by pooneyy** [reply]
windows上的话我建议你用[v2rayN](https://github.com/2dust/v2rayN)，v2rayA适合服务器使用

**Comment by niyu-bsp** [comment]
删除安装目录，然后重新安装

**Comment by zhigao1986** [comment]
我的群晖Docker  安装V2raya 忘记账号，多次测试 成功重置账号密码  SSH登录群晖 命令如下：“docker exec -it mzz2017-v2raya1 v2raya --reset-password ”     其中“mzz2017-v2raya1” 为v2ray docker 容器名称，注意修改成自己的

**Reply by pooneyy** [reply]
显示
```
Config directory: /etc/v2raya
Resetting password...
If no response for a long time, please stop other v2rayA instances and try again.
```
怎么办？

**Reply by waaall** [reply]
先关上服务：
sudo systemctl stop v2raya.service 
然后再重置密码就可以了：apt 安装的用 sudo v2raya --reset-password

**Reply by zonglin-tian** [reply]
> 显示
> 
> ```
> Config directory: /etc/v2raya
> Resetting password...
> If no response for a long time, please stop other v2rayA instances and try again.
> ```
> 
> 怎么办？

sudo rm -rf /etc/v2raya

# 因为是通过 sudo 安装的, 配置文件在 /etc 下, 所以 得删除这个文件
